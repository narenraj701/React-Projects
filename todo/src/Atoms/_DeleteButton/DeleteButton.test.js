import React from 'react';
import { render,screen } from '@testing-library/react';
import {create} from 'react-test-renderer'
import Delete from './DeleteButton'

test("Match the snapshot - DeleteButton",() => {
    const input=create(<Delete />);
    expect(input.toJSON()).toMatchSnapshot();
})
test("Delete Button is present", () => {
    render(<Delete />)
    let button = screen.getByRole("button");
    expect(button).toBeInTheDocument()
});
test("Button has text Delete", () => {
    render(<Delete />)
    let button = screen.getByRole("button");
    expect(button).toHaveTextContent("Delete")
});



