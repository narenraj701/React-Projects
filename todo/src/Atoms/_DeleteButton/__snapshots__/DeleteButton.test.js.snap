// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match the snapshot - DeleteButton 1`] = `
<button
  className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedSecondary"
  disabled={false}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  type="button"
>
  <span
    className="MuiButton-label"
  >
    Delete
  </span>
</button>
`;
